src/ObjectStore/v1/Models/StorageObject.php

```diff
@@ -35,10 +35,10 @@ class StorageObject extends OperatorResource implements Creatable, Deletable, Ha
     /** @var string */
     public $contentType;
 
-    /** @var int */
+    /** @var \DateTimeImmutable */
```

Comment:
This typehint cant be right. This is a `Content-Length` header, it should probably be a string (according to API spec)

https://developer.openstack.org/api-ref/object-store/#get-object-content-and-metadata